lib/appc/annotation.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 The appc Authors
```

Comment:
why did this get renamed to the singular annotation but dependencies stayed plural? :stuck_out_tongue: